tubular/edx_api.py

```diff
@@ -234,8 +273,10 @@ def retirement_retire_forum(self, learner):
         params = {'data': {'username': learner['original_username']}}
         try:
             with correct_exception():
-                return self._client.api.discussion.v1.accounts.retire_forum.post(**params)
-        except HttpNotFoundError:
+                api_url = self.get_api_url('api/discussion/v1/accounts/retire_forum')
+                results = self._request('POST', api_url, data=params).json()
+                return results
+        except HTTPError:
```

Comment:
There is only the True return value for 404 status_code before. Is it ok to return True on any 4xx and 5xx errors now?